ui/pages/settings/deprecated-network-modal/DeprecatedNetworkModal.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  Box,
+  ButtonLink,
+  ButtonPrimary,
+  ButtonPrimarySize,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from '../../../components/component-library';
+import {
+  Display,
+  FontWeight,
+  TextAlign,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import ZENDESK_URLS from '../../../helpers/constants/zendesk-url';
+
+interface Props {
```

Comment:
Nit: The Design System team uses more descriptive type names like `DeprecatedNetworkModalProps` -- that could be nice here.